src/OrbitGl/App.cpp

```diff
@@ -262,6 +265,47 @@ ErrorMessageOr<std::optional<std::filesystem::path>> GetOverrideSymbolFileForMod
   return symbol_file_path;
 }
 
+// Searches through an inout modules list for a module which paths contains path_substring. If one
+// is found the module is erased from the modules list and returned. If not found, std::nullopt is
+// returned.
+std::optional<const ModuleData*> FindAndEraseModuleByPathSubstringFromModuleList(
+    std::vector<const ModuleData*>& modules, std::string_view path_substring) {
```

Comment:
Nit: I would prefer `modules` to be a pointer to make it clearer to the user that this is making a change. (Same pattern as in the Abseil functions that mutate arguments). Not really requirement though. Feel free to ignore.

Reply:
done